docs/DeveloperGuide.adoc

```diff
@@ -819,14 +867,14 @@ _{More to be added}_
 (For all use cases below, the *System* is the `AddressBook` and the *Actor* is the `user`, unless specified otherwise)
 
 [discrete]
-=== Use case: Delete person
+=== Use case: Add a timetable entry on anyone's timetable
 
 *MSS*
 
-1.  User requests to list persons
-2.  AddressBook shows a list of persons
-3.  User requests to delete a specific person in the list
-4.  AddressBook deletes the person
+1. User requests to list employees
+2. AddressBook shows a list of employees
```

Comment:
Its probably best to rename "AddressBook" as this new app is not exactly an "Address Book".